pcdsdevices/spectrometer.py

```diff
@@ -247,6 +247,14 @@ class Mono(BaseInterface, Device):
                       doc='mirror pitch upstream encoder [urad]')
     g_pi_up_enc = Cpt(PytmcSignal, ':ENC:G_PI:02', io='i', kind='normal',
                       doc='grating pitch upstream encoder [urad]')
+    
+    # LED PWR
+    led_power_1 = Cpt(PytmcSignal, ':LED:01:PWR', io='io', kind='config',
+                    doc='LED power supply controls.')
```

Comment:
The tests on Travis are failing due to the formatting in the `led_power` lines. The `doc` needs to line up with `PytmcSignal` in the above line.
```suggestion
                      doc='LED power supply controls.')
```